admin-dev/themes/new-theme/scss/pages/_translation_page.scss

```
@@ -1,5 +1,321 @@
+$color-error: #c05c67 !default;
+$color-active: #25b9d7 !default;
+
 .export-translations {
   .export-translations-child {
     padding-left: 50px;
   }
 }
+
+.search-translation,
+.domains-tree {
+  .alerts {
+    margin-top: 1rem;
+  }
+
+  .col-offset {
+    margin-left: 1rem;
+  }
+
+  .hide {
+    display: none;
+  }
+}
+
+.breadcrumb {
+  small {
+    color: $gray-medium;
+  }
+}
+
+.total-remaining-translations {
+  color: $color-error;
+}
+
+.search-translation {
+  margin-bottom: 1.25rem;
+
+  .summary {
+    font-size: 0.875rem;
+    color: #363a41;
+  }
+
+  .search {
+    width: 100%;
+    margin-top: 1px;
+  }
+
+  .search-button {
+    margin-right: 0.625rem;
+  }
+
+  .alert {
+    float: left;
+    width: 42.63rem;
+    height: 3.438rem;
+    margin-top: 1.875rem;
+
+    i,
+    span {
+      float: left;
+    }
+
+    span {
+      margin-left: 2.5rem;
+    }
+  }
+}
+
+.domains-tree {
+  float: left;
+
+  hr {
+    height: 0.125rem;
+    margin-top: 0;
+    clear: both;
+    border-top: solid 1px #bbcdd2;
```

```suggestion
    border-top: solid 1px #bbcdd2;
```

Variables exists for this value